src/test/java/com/epam/task03/data/DataReaderTest.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.task03.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DataReaderTest {
+
+    private static final String PATH_TO_INPUT_FILE = "./src/test/resources/input.txt";
+    private static final List<String> VALID_DATA = Arrays.asList("1.0 0.0 3.0 0.0", "2.0 0.5 3.0 0.5");
+    private static final List<String> INVALID_DATA = Arrays.asList("1.0 0.0 3.0 0.0", "2.0 0.5 3.0 0.5", "");
+    private final DataReader reader = new DataReader();
+
+    @Test
+    public void testReadDataWithProvidedPathShouldSucceed() throws DataException {
+        List<String> actualData = reader.readData(PATH_TO_INPUT_FILE);
+        Assert.assertEquals(VALID_DATA, actualData);
+    }
+
+    @Test
+    public void testReadDataWhenFileEndsWithEmptyLineShouldFail() throws DataException {
```

Comment:
test should never fail (should always pass)
you should not compare with invalid data (only with valid)
negetive scenario would be to read non existing file and expect an exception @Test(expected =